### ISSUE TITLE
panic when gen grpc.pb.go from proto which has entproto.Service()

### ISSUE THREAD

**Issue by jony4** [ent/ent#2561]
- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/ent/ent/issues) of this repository and believe that this is not a duplicate.

## Current Behavior 😯

panic when generate grpc.pb.go, here is panic info

```shell
--entgrpc_out: template execution failed: template: to_proto.tmpl:60:17: executing "field_to_proto" at <newConverter .Field>: error calling newConverter: runtime error: invalid memory address or nil pointer dereference
generate.go:3: running "protoc": exit status 1
```

command:

```shell
/usr/local/Cellar/go/1.18.2/libexec/bin/go generate ent/proto/entpb/generate.go
```

## Expected Behavior 🤔

no panic.

## Steps to Reproduce 🕹

Steps:

1. gen proto files.
2. run command above to gen pb.go
3. panic
4.

More infos:

this is NOT OK

```go
func (Note) Annotations() []schema.Annotation {
	return []schema.Annotation{
		entproto.Message(),
		entproto.Service(),
	}
}
```

this is OK when I had commented entproto.Service()

```go
func (Note) Annotations() []schema.Annotation {
	return []schema.Annotation{
		entproto.Message(),
		// entproto.Service(),
	}
}
```

## Your Environment 🌎

| Tech        | Version |
| ----------- | ------- |
| Go          | 1.18.2  |
| Ent         | v0.10.2-0.20220521203114-1e05ccc28448   |
| Database    | MySQL   |
| Driver      | https://github.com/go-sql-driver/mysql        |

**Comment by rotemtam** [comment]
Hi @jony4 
Thanks for reaching out. 
I'm having trouble reproducing the issue based on the information you shared. 
Can you please use [ent/bug](https://github.com/ent/bug) to reproduce the issue? 
Thanks

**Comment by rotemtam** [comment]
Thanks for the thorough bug report @jony4.

I've managed to reproduce the issue locally and am working on a fix.

Will update

**Comment by rotemtam** [comment]
hey @jony4 ,

just merged a fix to this with https://github.com/ent/contrib/pull/322, if you can try again against ent/contrib@master

**Comment by jony4** [comment]
> hey @jony4 ,
> 
> just merged a fix to this with [ent/contrib#322](https://github.com/ent/contrib/pull/322), if you can try again against ent/contrib@master

Thanks for your job, I will try again later.
